spring-data-neo4j-rx/src/test/java/org/neo4j/springframework/data/integration/imperative/RepositoryWithADifferentDatabaseIT.java

```diff
@@ -48,9 +48,11 @@
 		PetRepository petRepository, Driver driver,
 		PersonWithRelationshipWithPropertiesRepository relationshipWithPropertiesRepository,
 		BidirectionalStartRepository bidirectionalStartRepository,
-		BidirectionalEndRepository bidirectionalEndRepository) {
```

Comment:
We should make a note to drop c'tor injection here and just inject the fields… Or on the methods, but this looks horrible.

Reply:
And I know I did this.